dll/cpl/sysdm/virtmem.c

```diff
@@ -456,7 +460,7 @@ OnSet(PVIRTMEM pVirtMem)
             /* Check the valid range of the maximum size */
             if (MaxSize < MinSize ||
                 MaxSize > pVirtMem->Pagefile[DriveIndex].FreeSize ||
-                MaxSize > 4095)
+                MaxSize > 4096)
```

Comment:
Please define a macro (such as `MAXSIZE`) for this magic number.

Reply:
No benefit (used once)